DZ3.2.py

```diff
@@ -0,0 +1,23 @@
+#Задача 18: Требуется найти в массиве A[1..N] самый близкий по величине элемент к заданному числу X. 
+#Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+#В последующих  строках записаны N целых чисел Ai. Последняя строка содержит число X
+
+#*Пример:*
+# 5
+#    1 2 3 4 5
+#    6
+#    -> 5
+
+N=int(input('Введите колличество элементов в массиве: '))
+numbers = [i for i in range(1,N+1)]
+print(numbers)
+
+
+X=int(input('Введите число Х: '))
+i=0
+if X>=N:
+    print('->', N)
+else:
+    i=X+1
+    print('->', i)
+
```

Comment:
Пожалуйста, не забывайте переводить к PEP8.
```suggestion
N = int(input('Введите колличество элементов в массиве: '))
numbers = [i for i in range(1, N + 1)]
print(numbers)

X = int(input('Введите число Х: '))
i = 0
if X >= N:
    print('->', N)
else:
    i = X + 1
    print('->', i)

```